lint-tasks.yml

```diff
@@ -128,6 +153,611 @@ tasks:
     internal: true
     cmd: "{{.ROOT_DIR}}/tools/yscope-dev-utils/lint-configs/symlink-cpp-lint-configs.sh"
 
+  check-cpp-static-diff:
```

Comment:
We should probably document the nuances of these tasks (e.g., that they write results to files). @coderabbitai, can you open an issue to add more detailed documentation for the lint tasks in this repo?